src/content/configuration/output.md

```diff
@@ -1085,6 +1085,21 @@ module.exports = {
 };
 ```
 
+## `output.enabledLibraryTypes`
+
+`array`
```

Comment:
array of strings?

Reply:
yes

How should , I add it
`array of strings` ?

Reply:
[string]